bot/resources/fun/top_1000_used_words.txt

```diff
@@ -0,0 +1,869 @@
+number
+occur
+offer
+office
+officer
+official
+often
+once
+only
+onto
+open
+operation
+opportunity
+option
+order
+organization
+other
+others
+outside
+over
+owner
+page
+pain
+painting
+paper
+parent
+part
+participant
+particular
+particularly
+partner
+party
+pass
+past
+patient
+pattern
+peace
+people
+perform
+performance
+perhaps
+period
+person
+personal
+phone
+physical
+pick
+picture
+piece
+place
+plan
+plant
+play
+player
+point
+police
+policy
+political
+politics
+poor
+popular
+population
+position
+positive
+possible
+power
+practice
+prepare
+present
+president
+pressure
+pretty
+prevent
+price
+private
+probably
+problem
+process
+produce
+product
+production
+professional
+professor
+program
+project
+property
+protect
+prove
+provide
+public
+pull
+purpose
+push
+python
+quality
+question
+quickly
+quite
+race
+radio
+raise
+range
+rate
+rather
+reach
+read
+ready
+real
+reality
+realize
+really
+reason
+receive
+recent
+recently
+recognize
+record
+reduce
+reflect
+region
+relate
+relationship
+religious
+remain
+remember
+remove
+report
+represent
+Republican
+require
+research
+resource
+respond
+response
+responsibility
+rest
+result
+return
+reveal
+rich
+right
+rise
+risk
+road
+rock
+role
+room
+rule
+safe
+same
+save
+scene
+school
+science
+scientist
+score
+season
+seat
+second
+section
+security
+seek
+seem
+sell
+send
+senior
+sense
+series
+serious
+serve
+service
+seven
+several
+shake
+share
+shoot
+short
+shot
+should
+shoulder
+show
+side
+sign
+significant
+similar
+simple
+simply
+since
+sing
+single
+sister
+site
+situation
+size
+skill
+skin
+small
+smile
+social
+society
+soldier
+some
+somebody
+someone
+something
+sometimes
+song
+soon
+sort
+sound
+source
+south
+southern
+space
+speak
+special
+specific
+speech
+spend
+sport
+spring
+staff
+stage
+stand
+standard
+star
+start
+state
+statement
+station
+stay
+step
+still
+stock
+stop
+store
+story
+strategy
+street
+strong
+structure
+student
+study
+stuff
+style
+subject
+success
+successful
+such
+suddenly
+suffer
+suggest
+summer
+support
+sure
+surface
+system
+table
+take
+talk
+task
+teach
+teacher
+team
+technology
+television
+tell
+tend
+term
+test
+than
+thank
+that
+their
+them
+themselves
+then
+theory
+there
+these
+they
+thing
+think
+third
+this
+those
+though
+thought
+thousand
+threat
+three
+through
+throughout
+throw
+thus
+time
+today
+together
+tonight
+total
+tough
+toward
+town
+trade
+traditional
+training
+travel
+treat
+treatment
+tree
+trial
+trip
+trouble
+true
+truth
+turn
+type
+under
+understand
+unit
+until
+upon
+usually
+value
+various
+very
+victim
+view
+visit
+voice
+vote
+wait
+walk
+wall
+want
+watch
+water
+wear
+week
+weight
+well
+west
+western
+what
+whatever
+when
+where
+whether
+which
+while
+white
+whole
+whom
+whose
+wide
+wife
+will
+wind
+window
+wish
+with
+within
+without
+woman
+wonder
+word
+work
+worker
+world
+worry
+would
+write
+writer
+wrong
+yard
+yeah
+year
+young
+your
+yourself
```

Comment:
This is no longer 1000 words haha

Reply:
I know! I believe @Shom770 was supposed to rename it to `hangman_word_list.txt` but never did.

Reply:
I thought I renamed it, does git not pick up file name changes?

Reply:
Nope, I guess not. I still see it as `top_1000_used_words.txt`.

Reply:
> Nope, I guess not. I still see it as `top_1000_used_words.txt`.

It is named `hangman_words.txt` if you go to my fork and go under `bot/resources/fun`

Reply:
Huh, you're right, @Shom770. Your fork does say `hangman_words.txt`. Then why does the new name not show up here then?

Reply:
This is outdated and has been fixed now.